dlp/google/cloud/dlp_v2/gapic/dlp_service_client.py

```diff
@@ -1769,10 +1769,14 @@ def list_dlp_jobs(
                    -  ``inspected_storage`` - DATASTORE\|CLOUD\_STORAGE\|BIGQUERY
                    -  ``trigger_name`` - The resource name of the trigger that created
                       job.
+                   -  'end\_time\` - Corresponds to time the job finished.
```

Comment:
I'm not sure if:
* This is a new filter OR
* They are just updating the documentation to say it is possible.

Will investigate.

Reply:
Left a comment asking for clarification in b/251716150

Reply:
@busunkim96 Ping?